apps/web/src/app/[orgShortCode]/_components/sidebar-content.tsx

```diff
@@ -0,0 +1,425 @@
+'use client';
+
+import useLoading from '@/src/hooks/use-loading';
+import { cn, generateAvatarUrl, getInitials } from '@/src/lib/utils';
+import { useGlobalStore } from '@/src/providers/global-store-provider';
+import { Button } from '@/src/components/shadcn-ui/button';
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage
+} from '@/src/components/shadcn-ui/avatar';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+  DropdownMenuSub,
+  DropdownMenuSubTrigger,
+  DropdownMenuSubContent,
+  DropdownMenuPortal,
+  DropdownMenuGroup
+} from '@/src/components/shadcn-ui/dropdown-menu';
+
+import {
+  Check,
+  AddressBook,
+  SignOut,
+  ChatCircle,
+  MoonStars,
+  Gear,
+  SpinnerGap,
+  Sun,
+  CaretRight,
+  CaretLeft,
+  SquaresFour,
+  Shield,
+  CaretUp,
+  Book,
+  QuestionMark,
+  MapPin,
+  Activity,
+  Megaphone,
+  PushPin,
+  CaretDoubleLeft,
+  CaretUpDown,
+  Plus,
+  Palette,
+  Monitor,
+  Question,
+  Paperclip,
+  User
+} from '@phosphor-icons/react';
+import { env } from 'next-runtime-env';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { useMemo } from 'react';
+import { toast } from 'sonner';
+import { useQueryClient } from '@tanstack/react-query';
+import React from 'react';
+import { SidebarNavButton } from './sidebar-nav-button';
+import { useTheme } from 'next-themes';
+import { sidebarSubmenuOpenAtom } from './atoms';
+import { useAtom } from 'jotai';
+import { usePreferencesState } from '@/src/stores/preferences-store';
+import {
+  ToggleGroup,
+  ToggleGroupItem
+} from '@/src/components/shadcn-ui/toggle-group';
+
+export default function SidebarContent() {
+  const orgShortCode = useGlobalStore((state) => state.currentOrg.shortCode);
+  return (
+    <div
+      className={cn(
+        'bg-slate-3 border-slate-5 z-40 flex h-full w-full min-w-56 resize-x flex-col items-start gap-4 rounded-2xl border p-2'
+      )}>
+      <OrgMenu />
+      <div
+        className={cn(
+          'flex w-full grow flex-col items-start justify-start gap-4 p-0'
+        )}>
+        <div className="flex w-full flex-col gap-0 p-0">
+          <span className="text-slate-10 p-1 text-[10px] font-semibold">
+            SPACES
+          </span>
```

Comment:
call it `Spaces` and add `uppercase` class to the span, its better for accessibility

Reply:
good point!